services/tables/src/main/java/com/linkedin/openhouse/tables/config/MainApplicationConfig.java

```diff
@@ -52,14 +53,23 @@ public class MainApplicationConfig extends BaseApplicationConfig {
 
   @Autowired StorageManager storageManager;
 
+  @Bean
+  public UserTableApi provideApiInstance() {
+    return new UserTableApi(getHtsConfiguredApiClient());
+  }
+
+  @Bean
+  public ToggleStatusApi provideToggleApiInstance() {
```

Comment:
should we introduce a new configuration file specific to feature toggle?

Reply:
This is the only new addition for the feature toggle, don't think we need to consider a separate config. 
And, this is a HTS related config so makes sense to be part of mainAppConfig?

Reply:
Can we add comments around how these beans are used. 
nit: The reason for asking a separate file was to make it easier for us to lift and shift this FeatureGate to other services if need be (jobs-service etc). ok to skip if unnecessary efforts.

Reply:
let's revisit that when feature toggle is needed to be in other services, because that might be a bigger change beyond spring boot-app config object.